index.js

```diff
@@ -52,8 +52,8 @@ app.get("/folders", async (req, res, next)=>{
 	}));
 });
 
-app.get("/icons/:size/:type/:name", async (req, res, next)=>{
```

Comment:
It's fine, but lets preserve old routes as well ( for backward compatibility )
So calls to /icons/size/type/name will call getIconURL with empty skin value

Reply:
done